app/services/organizational_chart_service.rb

```diff
@@ -0,0 +1,70 @@
+class OrganizationalChartService
+  def initialize company: nil
+    @company = company
+  end
+
+  def perform
+    organizational_chart_company
+  end
+
+  private
+
+  attr_reader :company
+
+  def organizational_chart_company
+    company_id = "company_id_#{company.id}"
+    data_rows_company = []
+    data_rows_company.append(company_data(company_id, company))
+    company.offices.each do |office|
+      data_rows_company.append(office_data(office, company_id))
+      office.users.each do |user|
+        data_rows_company.append(user_data(user))
+      end
+    end
+    data_table_company data_rows_company
+  end
+
+  def data_table_company data_rows_company
+    data_table = GoogleVisualr::DataTable.new
+    data_table.new_column("string", "name")
```

Comment:
```suggestion
    data_table.new_column :string, :name
```
Như này được không á